docs/a320-simvars.md

```diff
@@ -1588,3 +1593,26 @@
 - A32NX_ENGINE_TANK_OIL:{index}
     - Number (quarts)
     - Total engine {index} oil quantity in the oil tank
+
+## Air Conditioning / Pressurisation / Ventilation
```

Comment:
We should think about prefixing those with `PRESS`. Something like `A32NX_AUTO_LANDING_ELEVATION` can be confused with autoland.

Reply:
@davidwalschots also suggested that a few weeks ago. Since it's two of you now, I've changed all the new variables (except for the local QNH) to be prefixed by `PRESS`.